defaults/main.yml

```diff
@@ -11,7 +11,8 @@ openhpc_drain_timeout: 86400
 openhpc_resume_timeout: 300
 openhpc_retry_delay: 10
 openhpc_job_maxtime: 24:00:00
-openhpc_config: "{{ openhpc_extra_config | default({}) }}"
+openhpc_slurm_configless: false
+openhpc_config: "{{ {'SlurmctldParameters':['enable_configless']} if openhpc_slurm_configless else {} }}"
```

Comment:
As discussed, not doing that as the `_default` and `_extra` suffixes are used in the appliance. However the "configless" logic has been reversed so that `openhpc_config` is now simple.